src/alfasim_score/units.py

```diff
@@ -14,3 +14,4 @@
 TEMPERATURE_UNIT = "degC"
 GAS_OIL_RATIO_UNIT = "sm3/sm3"
 HEAT_TRANSFER_COEFFICIENT_UNIT = "W/m2.K"
+NONE_UNIT = "unitless"
```

Comment:
We usually call `dimensionless`

Reply:
https://github.com/ESSS/alfasim-sdk/blob/8fa83bdc5b2fcb0fae5d06215822ae344295464f/src/alfasim_sdk/_internal/alfacase/case_description.py#L2134

Reply:
"dimensionless" is the category of the "unitless" "unit".

Reply:
Yeah, got it. But I think I could use only `DIMENSIONLESS` without the `_UNIT`.

> https://github.com/ESSS/alfasim-sdk/blob/8fa83bdc5b2fcb0fae5d06215822ae344295464f/src/alfasim_sdk/_internal/alfacase/case_description.py#L2134

 I mentioned the category because the link points to row which are configuring the category as `dimensionless`.

Reply:
done